src/style-spec/function/index.js

```diff
@@ -90,6 +90,9 @@ function createFunction(parameters: FunctionParameters, propertySpec: StylePrope
                 throw new Error(f.zoomCurve ? f.zoomCurve.error : 'Invalid zoom expression');
             }
         }
+
+        // useful for debugging, especially for converted stop functions
+        f.rawExpression = expr;
```

Comment:
should we conditionally save this property (i.e. add a debug argument/option to `createFunction`)? not sure it would make a big difference beyond clarity on what `rawExpression` is used for.

Reply:
Yeah, I considered that briefly, but AFAIK the current build process doesn't have an easy way to do `if (DEBUG BUILD) {...}`.  We could use `envify`, which we're already using for the benchmarks, but since this seems pretty harmless, I thought maybe it wasn't worth adding a whole transform to the browserify build...

Reply:
yeah let's go with not over-complicating things 👌